packages/blockchain-wallet-v4-frontend/src/modals/OrderMyCard/index.tsx

```diff
@@ -1,34 +1,77 @@
-import React, { useState } from 'react'
+import React, { useEffect, useState } from 'react'
 import { connect, ConnectedProps } from 'react-redux'
-import { compose } from 'redux'
+import { bindActionCreators, compose, Dispatch } from 'redux'
 
 import Flyout, { duration } from 'components/Flyout'
+import { actions, selectors } from 'data'
 import { ModalName } from 'data/modals/types'
+import { RootState } from 'data/rootReducer'
 import modalEnhancer from 'providers/ModalEnhancer'
 
+import { Loading, LoadingTextEnum } from '../components'
 import { ModalPropsType } from '../types'
 import CreateCardStep from './CreateCardStep'
+import FailedCreationStep from './FailedCreationStep'
+import { LoadingWrapper } from './model'
+import { getData } from './selectors'
 
 const OrderMyCard = (props: Props) => {
   const [show, setShow] = useState(true)
 
+  // MVP only one product will be available, for V1 there will be a product selector in the UI
+  const { productCode } = props.products[0]
```

Comment:
you could double destructure this like `const [{ productCode }] = props.products` but it's not a blocker.

Reply:
good catch. Done